pkg/controller/masteruserrecord/masteruserrecord_controller.go

```diff
@@ -95,6 +101,111 @@ func (r *ReconcileMasterUserRecord) Reconcile(request reconcile.Request) (reconc
 		return reconcile.Result{}, err
 	}
 
-	// Do nothing for now
+	for _, account := range userRecord.Spec.UserAccounts {
+		err = r.ensureUserAccount(reqLogger, account, userRecord)
+		if err != nil {
+			reqLogger.Error(err, "unable to synchronize with member UserAccount")
+			return reconcile.Result{}, err
+		}
+	}
+
 	return reconcile.Result{}, nil
```

Comment:
should we set status=`Ready+True` here ??

Reply:
not yet - this is part of the synchronization pr